ci-scripts/run-tests.sh

```diff
@@ -92,19 +92,6 @@ set +e
 npm test
 result=$?
 
-set +x
```

Comment:
I'm curious, why don't we want to see the PFE logs if a test failed?

Reply:
I added back the test to stop and remove the containers so the container doesn't exist by this point. The logs were never useful anyway